-------------------- Log Start | Thu Jul 18 14:31:26 2024 --------------------
platform: Linux-6.6.28+rpt-rpi-2712-aarch64-with-glibc2.36
data_path: /home/blackaxis/Ender3_data
is_default_data_path: False
config_file: /home/blackaxis/Ender3_data/config/moonraker.conf
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: 5f9d5402db9a40248e3d1a5c01ed37ed
unix_socket_path: /home/blackaxis/Ender3_data/comms/moonraker.sock
software_version: v0.8.0-379-g346a3d7
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/blackaxis/Ender3_data/logs/moonraker.log
python_version: 3.11.2 (main, Mar 13 2023, 12:18:29) [GCC 12.2.0]
launch_args: /home/blackaxis/moonraker-env/bin/python /home/blackaxis/moonraker/moonraker/moonraker.py -d /home/blackaxis/Ender3_data
msgspec_enabled: False
uvloop_enabled: False

#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7126
klippy_uds_address = /home/blackaxis/Ender3_data/comms/klippy.sock

[authorization]
trusted_clients = 
	10.0.0.0/8
	127.0.0.0/8
	169.254.0.0/16
	172.16.0.0/12
	192.168.0.0/16
	FE80::/10
	::1/128
cors_domains = 
	*.lan
	*.local
	*://localhost
	*://localhost:*
	*://my.mainsail.xyz
	*://app.fluidd.xyz

[octoprint_compat]

[history]

[update_manager]
channel = dev
refresh_interval = 168

[update_manager mainsail-config]
type = git_repo
primary_branch = master
path = ~/mainsail-config
origin = https://github.com/mainsail-crew/mainsail-config.git
managed_services = klipper

[update_manager canboot]
type = git_repo
origin = https://github.com/Arksine/Katapult.git
path = ~/Katapult
is_system_service = False

[update_manager moonraker-telegram-bot]
type = git_repo
path = ~/moonraker-telegram-bot
origin = https://github.com/nlef/moonraker-telegram-bot.git
env = ~/moonraker-telegram-bot-env/bin/python
requirements = scripts/requirements.txt
install_script = scripts/install.sh

[update_manager fluidd-config]
type = git_repo
primary_branch = master
path = ~/fluidd-config
origin = https://github.com/fluidd-core/fluidd-config.git
managed_services = klipper

[update_manager fluidd]
type = web
channel = stable
repo = fluidd-core/fluidd
path = ~/fluidd

[update_manager mobileraker]
type = git_repo
path = /home/blackaxis/mobileraker_companion
origin = https://github.com/Clon1998/mobileraker_companion.git
primary_branch = main
managed_services = mobileraker
env = /home/blackaxis/mobileraker-env/bin/python
requirements = scripts/mobileraker-requirements.txt
install_script = scripts/install.sh

#################################################################
All Configuration Files:
/home/blackaxis/Ender3_data/config/moonraker.conf
#################################################################
Loading Sqlite database provider. Sqlite Version: 3.40.1
Unsafe Shutdown Count: 58

System Info:

***python***
  version: (3, 11, 2, 'final', 0)
  version_string: 3.11.2 (main, Mar 13 2023, 12:18:29) [GCC 12.2.0]

***cpu_info***
  cpu_count: 4
  bits: 64bit
  processor: aarch64
  cpu_desc: 
  serial_number: dddf145f114e1876
  hardware_desc: 
  model: Raspberry Pi 5 Model B Rev 1.0
  total_memory: 8245712
  memory_units: kB

***sd_info***
  manufacturer_id: 6f
  manufacturer: Unknown
  oem_id: 0303
  product_name: SDABC
  product_revision: 1.0
  serial_number: aa0001c7
  manufacturer_date: 5/2023
  capacity: 58.3 GiB
  total_bytes: 62550179840

***distribution***
  name: Debian GNU/Linux 12 (bookworm)
  id: debian
  version: 12
  version_parts: {'major': '12', 'minor': '', 'build_number': ''}
  like: 
  codename: bookworm
  release_info: {}
  kernel_version: 6.6.28+rpt-rpi-2712

***virtualization***
  virt_type: none
  virt_identifier: none

***network***

***canbus***

***Allowed Services***
  klipper_mcu
  webcamd
  MoonCord
  KlipperScreen
  moonraker-telegram-bot
  moonraker-obico
  sonar
  crowsnest
  octoeverywhere
  ratos-configurator
  mobileraker
Module libcamera unavailble, import failed

Systemd unit moonraker-Ender3.service:
unit_name: moonraker-Ender3.service
is_default: False
manager: systemd
Properties:
**Requires=['system.slice', 'sysinit.target', '-.mount', 'network-online.target']
**After=['network-online.target', 'basic.target', 'system.slice', 'sysinit.target', '-.mount', 'systemd-journald.socket']
**SupplementaryGroups=['moonraker-admin']
**EnvironmentFiles=/home/blackaxis/Ender3_data/systemd/moonraker.env
**ExecStart=/home/blackaxis/moonraker-env/bin/python $MOONRAKER_ARGS
**WorkingDirectory=/home/blackaxis/moonraker
**FragmentPath=/etc/systemd/system/moonraker-Ender3.service
**Description=API Server for Klipper SV1 Ender3
**User=blackaxis

Systemd unit klipper-Ender3.service:
unit_name: klipper-Ender3.service
is_default: False
manager: systemd
Properties:
**Requires=['sysinit.target', 'system.slice', '-.mount']
**After=['basic.target', 'system.slice', 'sysinit.target', 'network-online.target', 'systemd-journald.socket', '-.mount']
**SupplementaryGroups=[]
**EnvironmentFiles=/home/blackaxis/Ender3_data/systemd/klipper.env
**ExecStart=/home/blackaxis/klippy-env/bin/python $KLIPPER_ARGS
**WorkingDirectory=/home/blackaxis/klipper
**FragmentPath=/etc/systemd/system/klipper-Ender3.service
**Description=Klipper 3D Printer Firmware SV1
**User=blackaxis
Multiple instances of Moonraker have the update manager enabled.
moonraker-Ender5Plus:2c83ed81882d4673aaad84dbff75b7c8:874
moonraker-Spare:cf1bd642409e4c7eb69cbf6a6650e64d:875
moonraker-Ender3:5f9d5402db9a40248e3d1a5c01ed37ed:873
Git Repo moonraker: No warnings detected
Git Repo klipper: No warnings detected
Git Repo mainsail-config: No warnings detected
Git Repo canboot: No warnings detected
Git Repo moonraker-telegram-bot: No warnings detected
Git Repo fluidd-config: No warnings detected
Git Repo mobileraker: No warnings detected
Klipper Version: v0.12.0-258-g9318901f
2024-07-18 10:31:26,096 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.0.180
2024-07-18 10:31:26,316 [application.py:prepare()] - Upload Request Received from 192.168.0.180
Content-Type: multipart/form-data; boundary=------------------------2ZwtyG6Lrv6WXxWy3saOQz
2024-07-18 10:31:28,421 [application.py:post()] - Processing Uploaded File: Din_Rail_Mellow_Super_8_MB_Mount_1h49m_0.20mm_240C_ASA_.gcode
2024-07-18 10:31:28,488 [shell_command.py:pipe_data_received()] - INFO:metadata:Object Processing is disabled

2024-07-18 10:31:28,531 [shell_command.py:_check_proc_success()] - Command (/home/blackaxis/moonraker-env/bin/python /home/blackaxis/moonraker/moonraker/components/file_manager/metadata.py -p /home/blackaxis/Ender3_data/gcodes -f "Din_Rail_Mellow_Super_8_MB_Mount_1h49m_0.20mm_240C_ASA_.gcode") successfully finished
2024-07-18 10:31:28,531 [klippy_apis.py:start_print()] - Requesting Job Start, filename = Din_Rail_Mellow_Super_8_MB_Mount_1h49m_0.20mm_240C_ASA_.gcode
2024-07-18 10:31:28,536 [application.py:log_request()] - 201 POST /api/files/local (192.168.0.180) [_TRUSTED_USER_] 2220.91ms
2024-07-18 10:31:28,556 [job_state.py:_status_update()] - Job Started: Din_Rail_Mellow_Super_8_MB_Mount_1h49m_0.20mm_240C_ASA_.gcode
2024-07-18 10:47:40,322 [klippy_apis.py:start_print()] - Requesting Job Start, filename = Din_Rail_Mellow_Super_8_MB_Mount_1h49m_0.20mm_240C_ASA_.gcode
2024-07-18 10:47:40,460 [job_state.py:_status_update()] - Job Started: Din_Rail_Mellow_Super_8_MB_Mount_1h49m_0.20mm_240C_ASA_.gcode
2024-07-18 11:02:34,387 [websockets.py:on_close()] - Websocket Closed: ID: 140735279271952 Close Code: 1001, Close Reason: None, Pong Time Elapsed: 2.86
2024-07-18 11:02:34,798 [application.py:log_request()] - 101 GET /websocket?token=XFA6AG7J2W7ABTSFARXSMHAFERD2SPNA (192.168.0.180) [_TRUSTED_USER_] 0.88ms
2024-07-18 11:02:34,798 [websockets.py:open()] - Websocket Opened: ID: 140735308518352, Proxied: False, User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/18.0 Safari/605.1.15, Host Name: 3dprinters.local
2024-07-18 11:02:34,810 [websockets.py:_handle_identify()] - Websocket 140735308518352 Client Identified - Name: fluidd, Version: 1.30.0-f3e4ac3, Type: web
2024-07-18 11:02:34,811 [file_manager.py:get_file_list()] - Updating File List <config>...
2024-07-18 11:42:21,892 [klippy_apis.py:pause_print()] - Requesting job pause...
2024-07-18 11:42:45,558 [klippy_apis.py:cancel_print()] - Requesting job cancel...
2024-07-18 12:02:53,681 [authorization.py:_prune_conn_handler()] - Trusted Connection Expired, IP: 192.168.0.180
2024-07-18 12:32:03,631 [klippy_apis.py:start_print()] - Requesting Job Start, filename = Din_Rail_Mellow_Super_8_MB_Mount_1h49m_0.20mm_240C_ASA_.gcode
2024-07-18 12:32:03,790 [job_state.py:_status_update()] - Job Started: Din_Rail_Mellow_Super_8_MB_Mount_1h49m_0.20mm_240C_ASA_.gcode
2024-07-18 12:52:22,581 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.0.180
2024-07-18 12:52:22,626 [websockets.py:on_close()] - Websocket Closed: ID: 140735308518352 Close Code: 1001, Close Reason: None, Pong Time Elapsed: 7.70
2024-07-18 12:52:22,978 [application.py:log_request()] - 101 GET /websocket?token=G7HUDTNAZTX7B43YRYMLEQBLH3HOE37N (192.168.0.180) [_TRUSTED_USER_] 0.38ms
2024-07-18 12:52:22,978 [websockets.py:open()] - Websocket Opened: ID: 140735279276816, Proxied: False, User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/18.0 Safari/605.1.15, Host Name: 3dprinters.local
2024-07-18 12:52:22,984 [websockets.py:_handle_identify()] - Websocket 140735279276816 Client Identified - Name: fluidd, Version: 1.30.0-f3e4ac3, Type: web
2024-07-18 12:52:22,984 [file_manager.py:get_file_list()] - Updating File List <config>...
2024-07-18 13:52:53,701 [authorization.py:_prune_conn_handler()] - Trusted Connection Expired, IP: 192.168.0.180
2024-07-18 14:11:27,004 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.0.180
2024-07-18 14:11:27,141 [application.py:log_request()] - 101 GET /websocket?token=64KYR3PUPZ6NHYC73B6VTK3PN5IUGEJ3 (192.168.0.180) [_TRUSTED_USER_] 0.85ms
2024-07-18 14:11:27,141 [websockets.py:open()] - Websocket Opened: ID: 140735308239568, Proxied: False, User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.5060.134 OBS/30.2.0 Safari/537.36, Host Name: 3dprinters.local
2024-07-18 14:11:27,170 [websockets.py:_handle_identify()] - Websocket 140735308239568 Client Identified - Name: fluidd, Version: 1.30.0-f3e4ac3, Type: web
2024-07-18 14:11:27,171 [file_manager.py:get_file_list()] - Updating File List <config>...
2024-07-18 15:12:53,714 [authorization.py:_prune_conn_handler()] - Trusted Connection Expired, IP: 192.168.0.180
2024-07-18 17:53:30,060 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.0.180
2024-07-18 17:53:30,314 [application.py:prepare()] - Upload Request Received from 192.168.0.180
Content-Type: multipart/form-data; boundary=------------------------JuTno8m3MRJAgQ4DH4M1Rb
2024-07-18 17:53:30,582 [application.py:post()] - Processing Uploaded File: Din_Rail_6X5mm_Stand_off_5m_0.20mm_240C_ASA_.gcode
2024-07-18 17:53:30,652 [shell_command.py:pipe_data_received()] - INFO:metadata:Object Processing is disabled

2024-07-18 17:53:30,686 [shell_command.py:_check_proc_success()] - Command (/home/blackaxis/moonraker-env/bin/python /home/blackaxis/moonraker/moonraker/components/file_manager/metadata.py -p /home/blackaxis/Ender3_data/gcodes -f "Din_Rail_6X5mm_Stand_off_5m_0.20mm_240C_ASA_.gcode") successfully finished
2024-07-18 17:53:30,686 [klippy_apis.py:start_print()] - Requesting Job Start, filename = Din_Rail_6X5mm_Stand_off_5m_0.20mm_240C_ASA_.gcode
2024-07-18 17:53:30,693 [application.py:log_request()] - 201 POST /api/files/local (192.168.0.180) [_TRUSTED_USER_] 378.77ms
2024-07-18 17:53:30,694 [job_state.py:_status_update()] - Job Started: Din_Rail_6X5mm_Stand_off_5m_0.20mm_240C_ASA_.gcode
2024-07-18 18:03:55,721 [websockets.py:on_close()] - Websocket Closed: ID: 140735308239568 Close Code: 1001, Close Reason: None, Pong Time Elapsed: 8.57
2024-07-18 18:57:53,748 [authorization.py:_prune_conn_handler()] - Trusted Connection Expired, IP: 192.168.0.180
